bob-common/src/bob_client.rs

```diff
@@ -209,6 +212,14 @@ pub mod b_client {
                 })
         }
 
+        pub fn reset_error_count(&self) {
+            self.err_count.store(0, Ordering::Relaxed);
+        }
+
+        pub fn increase_error(&self) -> usize {
```

Comment:
These 2 functions should not be public. The should be used within the functions of this struct: in `put`, `get`, `exist` and `delete`

Reply:
You can add a public function to read `self.err_count` value